vhdbuilder/packer/vhd-image-builder-arm64-gen2.json

```diff
@@ -311,6 +311,23 @@
         "sudo FEATURE_FLAGS={{user `feature_flags`}} BUILD_NUMBER={{user `build_number`}} BUILD_ID={{user `build_id`}} COMMIT={{user `commit`}} HYPERV_GENERATION={{user `hyperv_generation`}} CONTAINER_RUNTIME={{user `container_runtime`}} TELEPORTD_PLUGIN_DOWNLOAD_URL={{user `teleportd_plugin_download_url`}} ENABLE_FIPS={{user `enable_fips`}} SGX_INSTALL=True /bin/bash -ux /home/packer/install-dependencies.sh"
       ]
     },
+    {
+      "type": "file",
+      "source": "vhdbuilder/packer/list-images.sh",
+      "destination": "/home/packer/list-images.sh"
+    },
+    {
+      "type": "shell",
+      "inline": [
+        "sudo BUILD_NUMBER={{user `build_number`}} BUILD_ID={{user `build_id`}} COMMIT={{user `commit`}} HYPERV_GENERATION={{user `hyperv_generation`}} CONTAINER_RUNTIME={{user `container_runtime`}} ENABLE_FIPS={{user `enable_fips`}} /bin/bash -ux /home/packer/list-images.sh"
```

Comment:
Just a general question (since it's the case for other parts of packer too): is there a good way to avoid repeating commands across various packer build files?

Reply:
ideally yeah -- you can see cluster-api is a bit better at this -- you can have multiple json files for variables and merge those. the steps still need to be the same but good way to parameterize: https://github.com/kubernetes-sigs/image-builder/blob/master/images/capi/packer/azure/azure-sig.json

think @anujmaheshwari1 had some thoughts on this for the future